src/main/java/frc/robot/Constants.java

```diff
@@ -30,6 +30,16 @@ public static class ControllerConstants {
      * 12 (COD)
      */
     public static class PortConstants {
+
+        public static final int SHOOTER_FOLLOWER_MOTOR_ID           = 42; //THE MEANING OF LIFE :) FAKE ID, CHANGE LATER! 
+        public static final int SHOOTER_LEADER_MOTOR_ID             = 64; //THESE ARE ALL FAKE IDs
+        public static final int SHOOTER_FLAP_MOTOR_ID               = 23; 
+        public static final int SHOOTER_PIVOT_MOTOR_ID              = 54;
+        public static final int SHOOTER_FOLLOWER_ENCODER_ID         = 63; 
+        public static final int SHOOTER_LEADER_ENCODER_ID           = 34; 
```

Comment:
no need for an ID for the encoder in this situation - we are using the integrated ones